tools/PublishProject/Program.cs

```diff
@@ -80,69 +83,80 @@ public static int Main(string[] args)
                                     string rid,
                                     string framework)
         {
-            var runtimeString = Regex.Replace(rid, "(\\d|\\.)*-", "-");
+            var runtimeString = Regex.Replace(rid, @"(\d|\.)*-", "-");
+            var buildIdentifier = $"{runtimeString}-{framework}";
 
-            // Disable for now, while travis isn't working correctly.
-            if (runtimeString.Contains("osx-")) {
-                return;
+            // Rename framework name to mono for non-windows platforms
+            if (runtimeString.Contains("ubuntu-") || runtimeString.Contains("osx-"))
+            {
+                buildIdentifier.Replace("dnx451", "mono");
             }
             
-            // Simplify Ubuntu to Linux
-            runtimeString = runtimeString.Replace("ubuntu", "linux");
-            var buildIdentifier = $"{runtimeString}-{framework}";
-            // Linux + dnx451 is renamed to Mono
-            if (runtimeString.Contains("linux-") && framework.Equals("dnx451"))
-                buildIdentifier ="linux-mono";
-            // No need to package OSX + dnx451
-            else if (runtimeString.Contains("osx-") && framework.Equals("dnx451"))
```

Comment:
But is it needed to have both platforms, since it builds against mono? The previous packaging just had omnisharp-mono for both platforms (without OS designation).